doc/source/data/api/data_source.rst

```diff
@@ -0,0 +1,12 @@
+.. _data-source-api:
+
+Global configuration
```

Comment:
To minimize clutter in the sidebar and improve the discoverability of `TFXReadOptions`, could we co-locate this reference with `read_tfrecords` in the Input/Output page? I don't think we should place `TFXReadOptions` on a page by itself if it's only used with `read_tfrecords`

Reply:
totally